admin/script/laptop_data.js

```diff
@@ -0,0 +1,93 @@
+$(document).ready(function () {
+    // Fungsi untuk memuat data menu
+    function loadLaptopData() {
+        $.ajax({
+            type: "GET",
+            url: host + "laptop_read.php",
+            dataType: "json",
+            success: function (response) {
+                var datalaptop = response.body.data;
+
+                // Hapus data yang ada
+                $("#datalaptop").empty();
+
+                // Muat ulang data menu setelah penghapusan berhasil
+                for (var i = 0; i < datalaptop.length; i++) {
+                    $("#datalaptop").append(
+                        `
+                        <tr>
+                            <td>` +
+                        (i + 1) +
+                        `</td>
+                            <td>` +
+                        datalaptop[i].kode +
+                        `</td>
+                            <td>
+                            <img src="../assets/img/` +
```

Comment:
ini mengarah ke alamat BE donk, file nya kan di upload ke project backend @agungsyhptra